src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -372,31 +371,25 @@ const FixPhoneNumbers: React.FC<Props> = ({
                   <Box className={classes.defaultBox}>
                     <Typography>{t('Default Primary Source:')}</Typography>
 
-                    <NativeSelect
-                      input={
-                        <StyledInput
-                          inputProps={{
-                            'data-testid': 'source-select',
-                          }}
-                        />
-                      }
-                      className={classes.nativeSelect}
-                      value={defaultSource}
+                    <Select
+                      className={classes.select}
                       data-testid="source-select"
+                      value={defaultSource}
                       onChange={(event: React.ChangeEvent<HTMLSelectElement>) =>
                         handleSourceChange(event)
                       }
+                      size="small"
                     >
-                      <option value="MPDX" data-testid="source-option-mpdx">
+                      <MenuItem value="MPDX" data-testid="source-option-mpdx">
                         MPDX
-                      </option>
-                      <option
+                      </MenuItem>
+                      <MenuItem
                         value="DataServer"
                         data-testid="source-option-dataserver"
                       >
                         DataServer
-                      </option>
-                    </NativeSelect>
+                      </MenuItem>
+                    </Select>
                     <Button
```

Comment:
Right now this button turns white when you hover over it. If you remove the `buttonBlue` class it should look better by default.